src/celeritas/em/params/WentzelOKVIParams.cc

```diff
@@ -43,7 +43,7 @@ WentzelOKVIParams::from_import(ImportData const& data,
         if (!coulomb)
         {
             // Set the maximum scattering angle for Wentzel VI MSC
-            return constants::pi;
+            return real_type(constants::pi);
```

Comment:
Just a slight note that this PR uses both `real_type(...)` and `real_type{...}` which is 'fine' but vaguely inconsistent.

Reply:
and a 3rd spelling: `static_cast<real_type>(...)`